.github/workflows/protobuf-check.yml

```diff
@@ -10,31 +10,30 @@ jobs:
 
   protobuf-check:
     name: protobuf-check
+    environment: GitLab
     runs-on: ubuntu-latest
     steps:
 
       - name: checkout
         uses: actions/checkout@v2
 
-      - name: checkout-dfinity
-        uses: actions/checkout@v2
-        with:
-          repository: dfinity-lab/dfinity
-          token: ${{ secrets.GH_TOKEN }}
-          path: dfinity
+      - name: checkout-ic
+        run: |
+          pushd ${{ runner.temp }}
+          curl --header "PRIVATE-TOKEN: ${{ secrets.GITLAB_TOKEN }}" https://gitlab.com/dfinity-lab/public/ic/-/archive/master/ic-master.tar.gz | tar xz
+          mv ic-master ic
+          popd
```

Comment:
We can get the sources from https://github.com/dfinity/ic, it's the public mirror of https://gitlab.com/dfinity-lab/public/ic. This way we don't need a token.

Reply:
Ah, nice point! Will pull from public mirror instead.